--- tesseract-3.02.02/dict/permdawg.cpp.orig	2013-04-04 09:43:12.704916485 +0100
+++ tesseract-3.02.02/dict/permdawg.cpp	2013-04-04 09:43:50.824914257 +0100
@@ -205,7 +205,7 @@
           STRING word_str;
           word->string_and_lengths(&word_str, NULL);
           word_str += " ";
-          fprintf(output_ambig_words_file_, word_str.string());
+          fprintf(output_ambig_words_file_, "%s", word_str.string());
         }
         WERD_CHOICE *adjusted_word = word;
         WERD_CHOICE hyphen_tail_word(&getUnicharset());
