Index: tesseract-3.01/ccutil/errcode.cpp
===================================================================
--- tesseract-3.01.orig/ccutil/errcode.cpp
+++ tesseract-3.01/ccutil/errcode.cpp
@@ -75,7 +75,7 @@ const char *format, ...          //speci
                                  //no specific
     msgptr += sprintf (msgptr, "\n");
 
-  fprintf(stderr, msg);
+  fprintf(stderr, "%s", msg);
   /*if ((strstr (message, "File") != NULL) ||
     (strstr (message, "file") != NULL))
   else if ((strstr (message, "List") != NULL) ||
